source/JsMaterialX/test/codeExamples.spec.js

```diff
@@ -0,0 +1,156 @@
+import { expect } from 'chai';
+import { initMaterialX, getMtlxStrings } from './testHelpers';
+
+describe('Code Examples', () => {
```

Comment:
These tests contain the JavaScript code snippets from the `CodeExamples.md` document.